### ISSUE TITLE
Training and submit script

### ISSUE THREAD

**Issue by youmi-zym** [princeton-vl/RAFT-Stereo#27]
Hi @lahavlipson,
Thanks for your great work. From the readme, I know how to train your code on scene flow, but if I want to see the result on kitti benchmark and other datasets, seems there is no explicit script given for training and submitting. Can you also provide these? Thanks.

**Comment by lahavlipson** [comment]
We don't have scripts for fine-tuning and submitting to the benchmark.

However, it should be sufficient to run the [training command](https://github.com/princeton-vl/RAFT-Stereo#training) with the flags `--train_datasets kitti` and `--restore_ckpt models/raftstereo-sceneflow.pth` and `--num_steps 5000`

**Comment by youmi-zym** [comment]
Thanks, I'll take a try. So after training as you suggested, I can test on the kitti2015 test dataset and submit to the kitti2015 benchmark, am I right?

**Comment by lahavlipson** [comment]
Yes. Also set the image size to at least 320x1000 and the maximum learning rate to 1e-5

**Comment by youmi-zym** [comment]
Thanks, I'll try and update you asap.
